testclient/src/main/java/org/corfudb/testclient/SimpleTrace.java

```diff
@@ -0,0 +1,54 @@
+package org.corfudb.testclient;
+
+import lombok.extern.slf4j.Slf4j;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+public class SimpleTrace {
```

Comment:
why we need this guy if we can use dropwizard metrics for that

Reply:
I want a very light implementation to measure the Recorder overhead.